Formula/w/wasi-runtimes.rb

```diff
@@ -22,6 +22,7 @@ class WasiRuntimes < Formula
   depends_on "cmake" => :build
   depends_on "lld" => [:build, :test]
   depends_on "wasi-libc" => [:build, :test]
+  depends_on "wasm-component-ld" => [:build, :test]
```

Comment:
Looks like we need this so that toolchain features are correctly detected when configuring the libc++ build.